script/Ciel/Day 2/02_01_CIEL02_1_3.txt

```diff
@@ -361,7 +386,7 @@ This guy saw me and Ciel-senpai together and snuck up on us...!
 [sha:e59e6e3f46c6c13e843bd47ccc939e34493262b7]{
 -- Page 80, Offset 3801.
 -- 「有彦……おまえ、なんでそう人の邪魔をすることばかりに長けているんだ？　前世からの呪いなのか？　それともそういう生き物なのか？」
-"Arihiko... why are you so good at getting in people's way? Is it a curse from a previous life? Or are you just that kind of creature?"
+"Arihiko... why are you so good at getting in people's way? Is it a curse from a previous life? Or is this just the way you are?"
```

Comment:
I wonder if the 'curse' is actually Shiki saying that Arihiko cursed him specifically in a previous life.